src/comm.cpp

```diff
@@ -283,6 +302,7 @@ void game_loop_unix(Fd control) {
          * Poll all active descriptors.
          */
         FD_ZERO(&in_set);
+        FD_ZERO(&in_set);
```

Comment:
FD_ZERO on in_set twice?

Reply:
Oh...sometimes it doesn't work first time, and you have to do it again...
er....whoops no, just a typo :blush: